beater/beater.go

```diff
@@ -713,6 +713,15 @@ func (s *serverRunner) newFinalBatchProcessor(
 	})
 	outputType := monitoring.NewString(monitoring.Default.GetRegistry("libbeat.output"), "type")
 	outputType.Set("elasticsearch")
+	monitoring.NewFunc(monitoring.Default, "libbeat.output.bulk_requests", func(_ monitoring.Mode, v monitoring.Visitor) {
+		v.OnRegistryStart()
+		defer v.OnRegistryFinished()
+		stats := indexer.Stats()
+		v.OnKey("available")
+		v.OnInt(stats.AvailableBulkRequests)
+		v.OnKey("completed")
+		v.OnInt(stats.BulkRequests)
+	})
```

Comment:
I couldn't figure out how to add `libbeat.output.bulk_requests_available`, so I nested `available` in a `bulk_requests` object, thinking that in the future we might want to add more metrics to that. I added `completed` because `available` seemed kind of lonely, but it's a duplicate of `events.batches`, so i'm also happy to remove it.

Reply:
Long term I'd like to stop naming the metrics we have `libbeat.output.*`. I did that initially so the metrics would show up in Stack Monitoring and friends.

Seeing as this metric doesn't exist in libbeat at all, let's not add that prefix please. Maybe `output.elasticsearch.bulk_requests.*`?